ml_metadata/tools/docker_server/Dockerfile

```diff
@@ -31,6 +31,7 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
   unzip \
   software-properties-common \
   git \
+  openjdk-8-jdk
```

Comment:
```suggestion
  openjdk-8-jdk \
```

Reply:
Why this dependency is needed?

Reply:
we need this dependency because bazel binary for linux/ppc64le needs java.

Reply:
@XinranTang - does that clarify your question?